bedrock/firefox/templates/firefox/all.html

```diff
@@ -30,7 +30,7 @@
   sky firefox-all-{{ channel }} {% if channel == 'alpha' %}blueprint{% endif %}
 {% endblock %}
 
-{% block site_css %}
+{% block page_css %}
```

Comment:
As discussed in meeting I'm going to leave the page_css block in. We can make another bug for testing better performance.